docs/platform_management_plan/project_management.rst

```diff
@@ -181,4 +181,219 @@ meetings, where such topics are discussed and decided, see `Steering committees`
+of the submodule. The project leads and committers of the corresponding software module,
+circle or community will check the issue and in case they will accept it, they will
+take it over to one of their *GitHub Projects*. In case, the topic, that you raise in the issue has a big impact on the platform, you can be asked by the committers to raise a *Contribution Request*.
+
+*Contribution Request* is the central concept to introduce any important changes
+to the project. It is described in more details in *Contribution Request* (TODO: link).
+In general, everyone who wants to provide something new to the project, e.g. a new feature
+or a tool, should provide an exhaustive description, requirements and in some cases
+also initial draft of the architecture as part of the *Contribution Request*.
+The *Contribution Requests* are regularly reviewed in the *Technical lead circle*
+and then get either accepted or declined.
+
+After the *Contribution Request* was accepted, then the *Pull Request* with the
+*Contribution Request* gets merged. The corresponding *GitHub Issue* gets a reference to the
+newly defined saga and gets closed. The saga is at the beginning in the state *"Draft"*. Please be aware, that "status" of the tickets is modelled in *GitHub Project* as *GitHub Issues* do not provide the possibility to define additional states.
+
+The *Technical lead circle* is responsible for maintenance of the backlog with sagas,
+their prioritization and creation of the roadmap. Together with software module
+project leads and community leads in the "Committer circle" they go through the backlog, decide when and which saga should be implemented in which order and update it the roadmap accordingly.
+
+As soon as the saga was planned for implementation, its state is changed to *"Open"*.
+As next step, an GitHub issue of type *epic* is created as sub-issue of the saga
+and gets assigned to one of the *Communities* for *Feature Architecture Definition*.
+The state of the saga changes from "Open" to "In Specification".
+
+.. image:: _assets/saga_status_workflow.svg
+    :width: 900
+    :alt: Planning workflow
+    :align: center
+
+The members of the *Responsible Community* define feature architecture, feature requirements
+and also component requirements for every involved component of software modules. Depending on the feature scope, one of the cross-functional team can be requested to make a POC in the `incubation repository <https://eclipse-score.github.io/score/features/integration/index.html#incubation-repositories>`_. Finally, *Responsible Community* does the break down of the corresponding *saga* to the tickets that can be assigned to the individual software modules or *communities*.
+As most of the software modules will have their own separate repository,
+then the detailed tracking of their work will also happen inside of that repository.
+However, the corresponding saga will still have a sub-issue of type epic,
```

Comment:
better: "corresponding saga of the SCORE repository"?

Reply:
Fixed